server/src/test/java/org/spine3/server/stand/SubscriptionRecordShould.java

```diff
@@ -0,0 +1,142 @@
+                                                                         Given.TYPE);
+
+        final Project entityState = Project.getDefaultInstance();
+        final Any wrappedState = AnyPacker.pack(entityState);
+        final ProjectId redundantId = ProjectId.getDefaultInstance();
+
+        final boolean matchResult = matchingRecord.matches(Given.TYPE, redundantId, wrappedState);
+
+        assertTrue(matchResult);
+    }
+
+    @Test
+    public void fail_to_match_improper_type() {
+        final SubscriptionRecord notMatchingRecord = new SubscriptionRecord(Given.subscription(),
+                                                                            Given.target(),
+                                                                            Given.TYPE);
+
+        final Project entityState = Project.getDefaultInstance();
+        final Any wrappedState = AnyPacker.pack(entityState);
+        final ProjectId redundantId = ProjectId.getDefaultInstance();
+
+        final boolean matchResult = notMatchingRecord.matches(Given.OTHER_TYPE, redundantId, wrappedState);
+
+        assertFalse(matchResult);
+    }
+
+    @Test
+    public void fail_to_match_improper_target() {
+        final ProjectId nonExisitngId = ProjectId.newBuilder()
+                                                 .setId("never-existed")
+                                                 .build();
+
+        final SubscriptionRecord notMatchingRecord = new SubscriptionRecord(Given.subscription(),
+                                                                            Given.target(nonExisitngId),
+                                                                            Given.TYPE);
+
+        final Project entityState = Project.getDefaultInstance();
+        final Any wrappedState = AnyPacker.pack(entityState);
+        final ProjectId redundantId = ProjectId.getDefaultInstance();
+
+        final boolean matchResult = notMatchingRecord.matches(Given.TYPE, redundantId, wrappedState);
+
+        assertFalse(matchResult);
+    }
+
+    @Test
+    public void be_equal_if_has_same_subscription() {
+        final Subscription oneSubscription = Given.subscription();
+        final Subscription otherSubscription = Subscription.newBuilder()
+                                                           .setId("breaking-id")
+                                                           .build();
+
+        @SuppressWarnings("QuestionableName")
+        final SubscriptionRecord one = new SubscriptionRecord(oneSubscription,
+                                                              Given.target(),
+                                                              Given.TYPE);
+
+        final SubscriptionRecord similar = new SubscriptionRecord(otherSubscription,
+                                                                  Given.target(),
+                                                                  Given.TYPE);
+
```

Comment:
Please remove redundant lines here AND in similar cases (after aligned parameters). Code here is spacy enough.